docs/spec/Specifiers/Connection-Graph-Specifiers.adoc

```diff
@@ -92,12 +92,8 @@ unmatched.
 
 .. For each of the two port instance identifiers _I_ appearing in _C_:
 
-... The component instance named in _I_
-must be available in the enclosing topology,
-either through
-<<Specifiers_Component-Instance-Specifiers,direct specification>>
-or through
-<<Specifiers_Topology-Import-Specifiers,import>>.
+... _I_ must name an <<Instance-Member-Identifiers_Port-Instance-Identifiers,
+port instance id>> available in the enclosing topology.
```

Comment:
The concept of availability is now defined in the section on Topology Definitions, right? If so, we should provide a link there. Otherwise it's not clear what "available" means here.